Packs/Polygon/Integrations/Polygon/Polygon.py

```diff
@@ -339,7 +349,15 @@ def get_network_indicators(report):
                 score=0
             )
         )
-        ids.append(domain)
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=domain,
+            raw_response=res
```

Comment:
Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.